common/lib/capa/capa/inputtypes.py

```diff
@@ -525,9 +526,10 @@ def extract_choices(element, i18n):
                 choices.append((choice.get("name"), stringify_children(choice)))
             else:
                 if choice.tag != 'compoundhint':
-                    msg = u'[capa.inputtypes.extract_choices] {error_message}'.format(
+                    msg = Text('[capa.inputtypes.extract_choices] {error_message}').format(
                         # Translators: '<choice>' and '<compoundhint>' are tag names and should not be translated.
-                        error_message=_('Expected a <choice> or <compoundhint> tag; got {given_tag} instead').format(
+                        error_message=Text(
+                            _('Expected a <choice> or <compoundhint> tag; got {given_tag} instead')).format(
```

Comment:
@robrap @cahrens Do I need to use Text or HTML in a case like this?  The tags are included in the text as examples.